Added code_of_conduct.pt-br.md (#828)
ContributorCovenant/contributor_covenant: content/version/2/0/code_of_conduct.pt-br.md

+++
version = "2.0"
aliases = ["/version/2/0"]
+++

> # Código de Conduta para Colaborantes

Reviewer: Não encontrei a palavra "Colaborante" no meu dicionário; talvez seja melhor usar "colaborador"?

```suggestion
# Código de Conduta para Colaboradores
```

Author: O uso do termo era para tornar a escrita inclusiva (visto que "colaborador" e "colaboradores" se referem a um gênero específico). Mas neste caso eu concordo que podemos ir pela exceção e usar "colaboradores" que é um termo mais comum.

> # Código de Conduta para Colaboradores

Reviewer 2: Some suggestions about how to remove the gender of `colaboradores` here:

(replacing the "Contributor" by "Contribution")

```suggestion
# Código de Conduta para Colaboração
```

```suggestion
# Código de Conduta de Colaboração
```

```suggestion
# Código de Conduta para Pessoas Colaboradoras
```

Reviewer 3: I really like these suggestions!

Author: Nice! I sticked to the second option:

> Código de Conduta de Colaboração

## Nossa promessa

> Nós, como participantes, contribuintes e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.

Reviewer: - Contribuinte é quem paga imposto, taxa, etc.

```suggestion
Nós, como participantes, colaboradores e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Reviewer: (Talvez `lideranças` deva ser apenas `liderança`? Ou está no plural de propósito?)

Author: Alterei o termo para "colaboradores" aqui também para seguir no padrão.
O plural foi intencional, visto que na versão em inglês está assim ("We as members, contributors, and **leaders**"). Aqui foi apenas o caso de eu querer ser fiel ao texto original. O que você acha?

Reviewer 4: As duas formas estão corretas: "a liderança" refere-se ao grupo de pessoas que exercem a liderança. No contexto de organizações partidárias, é comum falar de "as lideranças". Isso pode se referir a diferentes instâncias, ou também às diferentes pessoas que formam a lidarança.

Author: Alterei para o singular em todos os locais.

> Nós, como participantes, contribuintes e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.

Reviewer 5: nit: the translation of the first sentence is not wrong but sounds a bit too literal. In Portuguese I believe that this might sounds better:

_"Como participantes, contribuintes e líderes, nós nos comprometemos..."_

Author: Sounds great, thank you. I changed the sentence like you suggested. The only thing I kept was the word "lideranças", as this is the word I chose for "leaders", in order to make the writing more inclusive since it's gender neutral.

> Como participantes, colaboradores e lideranças, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.

Reviewer 3: I saw that you changed "liderenças" to "liderança" in multiple places. Shouldn't you change it here too? 
```suggestion
Como participantes, colaboradores e liderança, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Author: In this particular case I left it intentionally, because it sounded weird with "nós" coming right after "liderança". But, come to think of it, and as someone else pointed out before, "liderança" could also mean a collective of leaders. So, for standards sake, I'm changing it to singular here too.

> Como participantes, colaboradores e lideranças, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.

Reviewer 2: Continue to think about the use of a gender-neutral word for contributors 

```suggestion
Como pessoas participantes, colaboradoras e líderes, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

TIL: "líder" is gender-neutral

Reviewer 2: Sincerely I don't remember if we could use "ênclise" here 😅 

```suggestion
Como participantes, colaboradores e lideranças, nós comprometemo-nos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Author: I did some research and my understanding is that we would only use "ênclise" if it was at the start of a sentence ("Comprometemo-nos a fazer ...") or if it was in the negative form (", não comprometemo-nos").

But nouns are really tricky when writing in portuguese, so I could be wrong 😅

Reviewer 3: I think we could use "comprometemo-nos" and get rid of "nós".

```suggestion
Como participantes, colaboradores e lideranças, comprometemo-nos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

But I can definitely see the value of keeping "nós" to keep the overall theme of collectiveness.

Author: This is great!

I think I'm keeping "nós", for this exact reason you said — to keep the overall theme of collectiveness. It was also a suggestion I got from other people before.

So for now it's going to stay like this:

> Como pessoas participantes, colaboradoras e líderes, nós nos comprometemos a fazer [...]

We could do a new round of reviews later, but this way looks good to me.

> Comprometemo-nos a agir e interagir de maneiras que contribuem para uma comunidade aberta, acolhedora, diversificada, inclusiva e saudável.

Reviewer 2: Subjuntivo? I don't remember the Portuguese rules as I used to know 😢 

```suggestion
Comprometemo-nos a agir e interagir de maneiras que contribuam para uma comunidade aberta, acolhedora, diversificada, inclusiva e saudável.
```

Reviewer 2: It seems `de maneira que` should be singular
My ref: http://g1.globo.com/educacao/blog/dicas-de-portugues/post/duvidas-dos-leitores-7.html

But the phrase sounds a little strange for me if I change it

```suggestion
Comprometemo-nos a agir e interagir de maneira que contribua para uma comunidade aberta, acolhedora, diversificada, inclusiva e saudável.
```

Author: Now that you've mentioned, it does sound a little strange! 😅
So I mixed both of your suggestions and now I'm sticking to this:

> Comprometemo-nos a agir e interagir de maneira que contribua para [...]

## Nossos padrões

Exemplos de comportamentos que contribuem para criar um ambiente positivo para a nossa comunidade incluem:

> * Demonstrar empatia e bondade com as outras pessoas

Reviewer 4: This and the following bullet points should end with a period.

Author: This is was a case where I wanted to stay true to the original english version of the text, where the bullet points don't end with a period.
What do you think? Should we add a period in every bullet list in the pt-br version anyway?

Reviewer 4: It's not important. It's OK to leave it as is.

> * Respeitar opiniões contrárias, pontos de vista e experiências

Reviewer 5: I believe this fits better the original text so _"contrárias"_ (differing) refer to all three more clearly: _"Respeitar opiniões, pontos de vista e experiências contrárias"_

Author: I changed it like you suggested :)

> * Dar e receber feedbacks construtivos de maneira graciosa

Reviewer 4: I'd use "respeitosa" instead of "graciosa". I can't think of a good translation for "graceful" in this sentence. "Respeitosa" is "respectful" which is a good free translation in my opinion.

Author: Tem razão. Eu adaptei do texto da versão 1 (que dizia "com graça"), mas concordo com você que assim fica melhor. Estou alterando para "respeitosa".

> * Assumir responsabilidade e pedir desculpas para as pessoas afetadas por nossos erros e aprender com a experiência

Reviewer 4: This sentence has three clauses and two "e" (and), which is not good. I'd replace the first "e" with a comma. Also it is usual in spoken language to say "pedir desculpas para" but the correct form for written language is "pedir desculpas a". So the full sentence would be:

"Assumir responsabilidade, pedir desculpas às pessoas afetadas por nossos erros e aprender com a experiência."

Author: Agreed, I'm making the changes you're suggesting,

* Focar no que é melhor não só para nós individualmente, mas para a comunidade em geral

Exemplos de comportamentos inaceitáveis incluem:

> * Uso de linguagem ou imagens sexuais e atenção ou avanço sexual de qualquer natureza

Reviewer 2: "Sexualizada" seems to be a better term here to include the "non-explicit content with a sexual intention"
I didn't know what could be "attention and advances" (in Portuguese and in English) but I can imagine

A suggestion to keep the phrase more similar to the English version
```suggestion
* Uso de linguagem ou imagens sexualizadas, atenção sexual ou avanços de qualquer natureza
```

Author: I agree with you, I don't think there's a direct and accurate way of translating this. So I'm going for something more commonly used in portuguese and now I'm sticking with this:

> Uso de linguagem ou imagens sexualizadas, bem como o assédio sexual ou de qualquer natureza

I think this represents well what the english version means.

* Comentários insultuosos/depreciativos e ataques pessoais ou políticos (*Trolling*)
* Assédio público ou privado
* Publicar informações particulares de outras pessoas, como um endereço de e-mail ou endereço físico, sem a permissão explícita delas
* Outras condutas que são normalmente consideradas inapropriadas em um ambiente profissional

## Aplicação das nossas responsibilities

> As lideranças da comunidade são responsáveis por clarificar e aplicar nossos padrões de comportamento aceitável e tomarão ações corretivas apropriadas e justas em resposta a qualquer comprotamento que considerarem impróprio, ameaçador, ofensivo ou problemático.

Reviewer 5: nit: _"clarificar"_ is not wrong, but maybe _"esclarecer"_ is more commonly used

Reviewer 5: regarding _"As lideranças"_, I believe either _"A liderança"_ (since its singular form already refers to a group) or _"Os líderes"_ could be a better fit.

ps: a friendly reminder that there are a few other usages of _"as lideranças"_ below. Depending on the decision you make here, adjust the remaining ones accordingly.

Reviewer 5: _"padrões de comportamento **aceitáveis**"_

Author: I'm still waiting for feedback from others on using plural or singular in "as lideranças". My main concern was to be faithful to the original text (which uses plural), but maybe I'm changing it to singular everywhere.

Reviewer 4: Achei legal que @james-sabel evitou "os líderes" para ser inclusivo. Para mim não faz muita diferença "as lideranças" ou "a liderança". As duas formas são corretas. Na dúvida eu aplico a navalha de Occam e usaria a forma singular (só porque tem menos letras, e parece mais formal)

Author: Alterei para o singular em todos os locais.

As lideranças da comunidade têm o direito e a responsabilidade de remover, editar ou rejeitar comentários, commits, códigos, edições na wiki, erros e outras contribuições que não estão alinhadas com este Código de Conduta e irão comunicar as razões por trás das decisões da moderação quando for apropriado.

## Escopo

Este Código de Conduta se aplica dentro de todos os espaços da comunidade e também se aplica quando uma pessoa estiver representando oficialmente a comunidade em espaços públicos.

> Exemplos de representação da nossa comunidade incluem usar um endereço de e-mail oficial, postar em contas oficiais de mídias sociais ou atuar como uma pessoa indicada a representante em um evento online ou offline.

Reviewer 4: "indicada a representante" sounds better as "indicada como representante"

Author: I changed it to "indicada como representante".

## Aplicação

Ocorrências de comportamentos abusivos, de assédio ou que sejam inaceitáveis por qualquer outro motivo poderão ser reportadas para a liderança da comunidade, responsável pela aplicação, via contato [INSERIR MÉTODO DE CONTATO].

> Todas as reclamações serão revisadas e investigadas imediatamente e de maneira justa.

Reviewer 2: Just to keep the consistency 🙂 
```suggestion

Todas as reclamações serão revisadas e investigadas imediatamente e de maneira justa.
```

Author: It does look better now. Thanks!

> Todas as lideranças da comunidade têm a obrigação de respeitar a privacidade e A segurança de quem reportar qualquer incidente.

Reviewer: ```suggestion
Todas as lideranças da comunidade têm a obrigação de respeitar a privacidade e a segurança de quem reportar qualquer incidente.
```

Author: Ops! :) Corrigido.

> Todas as lideranças da comunidade têm a obrigação de respeitar a privacidade e A segurança de quem reportar qualquer incidente.

Reviewer 4: "A segurança" -> "a segurança"

Author: Oops! :) Fixed it.

## Diretrizes de aplicação

> As lideranças da comunidade seguirão estas Diretrizes de Impacto na Comunidade para determinar as consequências de qualquer ação que considerarem violadora deste Código de Conduta:

Reviewer: Não seria melhor se fosse no singular?

```suggestion
A liderança da comunidade seguirá estas Diretrizes de Impacto na Comunidade para determinar as consequências de qualquer ação que considerarem violadora deste Código de Conduta:
```

Author: Como no comentário anterior, a minha intenção era ser fiel ao texto original em inglês (que está no plural).
O que você acha? Alteramos tudo para o singular?

Reviewer 4: Isso, depois de refletir, também prefiro o singular.

Author: Ótimo! Estou alterando para o singular em todos os locais, vou fazer mais uma revisão geral e envio o novo pull request em seguida.

### 1. Ação corretiva

**Impacto na comunidade**: Uso de linguagem imprópria ou outro comportamento considerado anti-profissional ou repudiado pela comunidade.

**Consequência**: Um aviso escrito e privado das lideranças da comunidade, esclarecendo a natureza da violação e uma explicação do motivo pelo qual o comportamento era impróprio. Um pedido de desculpas público poderá ser solicitado.

> ### 2. Advertêcia

Reviewer 5: typo: _"Advertê**n**cia"_

Author: Oops! Fixed it :)

> **Impacto na comunidade**: Uma violação por meio de um incidente único ou atitudes repetidas.

Reviewer 4: Starting with the article "Uma" is not helpful here. The sentence is clearer without it. As it is, it suggests a count: "One violation" which doesn't make sense in the context.

Author: Fixed it

> **Consequência**: Uma advertência com consequências para comportamento repetido. Não poderá haver interações com as pessoas envolvidas, incluindo interações não solicitadas com as pessoas que estiverem aplicando o Código de Conduta, por um período determinado. Isto inclui evitar interações em espaços da comunidade, bem como canais externos como as mídias sociais. A violação destes termos podem levar a um banimento temporário ou permanente.

Reviewer 4: Starting with the article "Uma" is not helpful here. The sentence is clearer without it.

Author: You are right, I changed it like you suggested.

### 3. Banimento Temporário

> **Impacto na comunidade**: Uma violação grave dos padrões da comunidade, incluindo o comportamento impróprio constante.

Reviewer 4: Another instance were removing the initial "Uma" will be an improvement.

Author: Fixed it here too :)

**Consequência**: Banimento temporário de qualquer tipo de interação ou comunicação pública com a comunidade por um determinado período. Estarão proibidas as interações públicas ou privadas com as pessoas envolvidas, incluindo interações não solicitadas com as pessoas que estiverem aplicando o Código de Conduta. A violação destes termos pode resultar em um banimento permanente.

### 4. Banimento Permanente

**Impacto na comunidade**: Demonstrar um padrão na violação das normas da comunidade, incluindo a persistência do comportamento impróprio, assédio a uma pessoa ou agressão ou depreciação a classes de pessoas.

**Consequência**: Banimento permanente de qualquer tipo de interação pública dentro da comunidade.

## Atribuição

Este Código de Conduta é adaptado do [Contributor Covenant][homepage],
versão 2.0, disponível em
https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.

As Diretrizes de Impacto na Comunidade foram inspiradas pela [Aplicação do código de conduta Mozilla](https://github.com/mozilla/diversity).

[homepage]: https://www.contributor-covenant.org

> Para obter respostas a perguntas comuns sobre este código de conduta, veja a página de Perguntas Frequentes (*FAQ*) em

Reviewer 2: maybe? 🤔 
```suggestion
Para obter respostas às perguntas comuns sobre este código de conduta, veja a página de Perguntas Frequentes (*FAQ*) em
```

Reviewer 3: "A perguntas" is correct here. :)

Author: Indeed, both ways are correct. And now y'all left me wondering hahaha

Author: Ok, so I'm keeping it with "a perguntas frequentes", as it addresses questions in a general manner. If we were to change it to "às perguntas frequentes", I think we'd be referring to specific/known questions.

### 3. Banimento Temporário

**Impacto na comunidade**: Violação grave dos padrões da comunidade, incluindo a persistência do comportamento impróprio.

**Consequência**: Banimento temporário de qualquer tipo de interação ou comunicação pública com a comunidade por um determinado período. Estarão proibidas as interações públicas ou privadas com as pessoas envolvidas, incluindo interações não solicitadas com as pessoas que estiverem aplicando o Código de Conduta. A violação destes termos pode resultar em um banimento permanente.

### 4. Banimento Permanente

**Impacto na comunidade**: Demonstrar um padrão na violação das normas da comunidade, incluindo a persistência do comportamento impróprio, assédio a uma pessoa ou agressão ou depreciação a classes de pessoas.

**Consequência**: Banimento permanente de qualquer tipo de interação pública dentro da comunidade.

## Atribuição

Este Código de Conduta é adaptado do [Contributor Covenant][homepage],
versão 2.0, disponível em
https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.

> As Diretrizes de Impacto na Comunidade foram inspiradas pela [Aplicação do código de conduta Mozilla](https://github.com/mozilla/diversity).

Reviewer 6: +As Diretrizes de Impacto na Comunidade foram inspiradas pela [Aplicação do código de conduta Mozilla](https://github.com/mozilla/diversity).
